src/box2d.rs

```diff
@@ -63,13 +63,21 @@ impl<T: Eq, U> Eq for Box2D<T, U> {}
 
 impl<T: fmt::Debug, U> fmt::Debug for Box2D<T, U> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Box2D({:?}, {:?})", self.min, self.max)
```

Comment:
this should be raised at a lower level, i.e. the "proper" way to go here is unnecessary verbose :/
I could imagine something like `write!(f, "Box2D({:~?}, {:~?})", self.min, self.max)` that would propagate the `f` parameters automatically. Could you raise that issue upstream?

Reply:
I agree with you, I think that the feature worth it though.

> Could you raise that issue upstream?

I don't have the steam to champion an RFC for this at the moment, I'm asking around to see if I could get away requesting a feature like that directly as a github issue.

In the process @nox pointed out that for simple Debug formats we can do: 

```rust
        f.debug_tuple("Box2D")
            .field(&self.min)
            .field(&self.max)
            .finish()
```

This is a bit nicer although it only works for `Debug` (no equivalent `Display` helpers), and won't work for Rect({size} at {origin}) as far as I can tell.

Would you prefer that I convert the debug impls that I can to this syntax or keep the more verbose but consistent way?

In any case I'd like to land this in some form of another without waiting for a potentital future version of the standard library.

Reply:
Converting the debug seems like an improvement to me, not a blocker to land this though.

Also, perhaps RFC isn't necessary the first step, but rather opening an issue saying that there is a problem here.

Reply:
I converted the debug impls to use debug_tuple where applicable and filed https://github.com/rust-lang/rust/issues/70710